### PR TITLE
AICLOSURE-1 - backup and restore details added to support docs

### PR DESCRIPTION
**Pull request by jonanswer**

### CODE REVIEW

File: documentation/support_docs.md [hunk 1]
```diff
@@ -94,24 +95,24 @@ FLIP implements a microservice-based architecture. The Image Service, Import Ser
 [TODO: List the Lambdas that we have defined and their purpose - dev team]
```
Comment (heyhaleema) [inline documentation/support_docs.md:95]: @jonanswer For the `TODO` comments in `[]`, could we remove these from the document/PR and raise corresponding issues instead? I feel like we'd be better able to track outstanding items and link these to future PRs, as well as keep the Jupyter Book clean whilst we're working on them.

File: documentation/support_docs.md [hunk 2]
```diff
@@ -213,9 +214,7 @@ All AWS components in the Central Hub will output their logs to Cloudwatch. This
 
 ## Data Catalogue
 
-The Data Catalogue was compiled from the data requirements of the models shortlisted for deployment out to FLIP. Data points were categorised into mandatory and optional - as many of the mandatory requirements as possible have been included in the initial data load, future enhancement to FLIP will include further data points.
-
-[Link to Data Catalogue](https://docs.google.com/spreadsheets/d/1wrFozOs4agLSqG4c0fS2YAsccVqQ4WMZWa2zVGOlJ0g/edit#gid=0)
+The [Data Catalogue](https://docs.google.com/spreadsheets/d/1wrFozOs4agLSqG4c0fS2YAsccVqQ4WMZWa2zVGOlJ0g/edit#gid=0) was compiled from the data requirements of the models shortlisted for deployment out to FLIP. Data points were categorised into mandatory and optional - as many of the mandatory requirements as possible have been included in the initial data load, future enhancement to FLIP will include further data points.
```
Comment (heyhaleema) [inline documentation/support_docs.md:217]: Can we move all external documents to (and therefore link in) this repo?

**Review by heyhaleema — CHANGES_REQUESTED** [review]
@jonanswer Sorry for the belated review! Just a couple minor requested edits, but this looks good to merge 👍

File: documentation/support_docs.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jonanswer) [reply documentation/support_docs.md]: TODO comments now removed, the content they referred to should be added in as it becomes available

File: documentation/support_docs.md (hunk 2 above)
Comment (jonanswer) [reply documentation/support_docs.md]: external docs moved in, all links repointed!

**Pull request merged**